### PR TITLE
Systems and components

### LINKED ISSUE #67: Frontend System/Components section
- [x] #58 
- [x] #59

### PR DESCRIPTION
**Pull request by Jesper276**
Closes #67 

### Changes/Additions:

- Made the right sidebar

### CODE REVIEW

File: src/lib/components/project/component/Component.svelte [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		<p>{name}</p>
+	</div>
+	<div>
+		<More_vert />
+	</div>
+</div>
+
+<style>
+	.left {
+		display: flex;
```
Comment (Jesper276) [inline src/lib/components/project/component/Component.svelte:26]:
```suggestion
		display: flex;
		align-items: center;
```

File: src/lib/components/project/system/System.svelte [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		<p>{name}</p>
+	</div>
+	<div>
+		<More_vert />
+	</div>
+</div>
+
+<style>
+	.left {
+		display: flex;
```
Comment (Jesper276) [inline src/lib/components/project/system/System.svelte:26]:
```suggestion
		display: flex;
		align-items: center;
```

**Review by BaBrixx — APPROVED** [review]
LGTM

**Review by SolarEarth37 — APPROVED** [review]
LGTM :)

**Pull request merged**